mwdb/resources/auth.py

```diff
@@ -327,6 +336,12 @@ def post(self):
         except NoResultFound:
             raise Forbidden("Invalid login or email address.")
 
+        if not user.has_rights(Capabilities.manage_profile):
+            raise Forbidden(
+                "Password recovery is disabled for this user. "
```

Comment:
Maybe "You are not allowed to recover the password. Ask..."?

Reply:
Solved :+1: